libs/langchain/langchain/llms/ollama.py

```diff
@@ -108,6 +111,7 @@ def _default_params(self) -> Dict[str, Any]:
                 "tfs_z": self.tfs_z,
                 "top_k": self.top_k,
                 "top_p": self.top_p,
+                "format": self.format,
```

Comment:
I think this needs to go outside `options` (same level as `prompt`):

https://github.com/jmorganca/ollama/blob/main/docs/api.md#request-json-mode